Fixed bug #33: display of Skype icon.

nihonjinrxs opened the pull request:
To test, you'll need to add the following to `/node_modules/resume_schema/resume.json` in the `basics.profiles` segment:

```javascript
      {
        "network": "Skype",
        "username": "myskypeusername"
      }
```

mudassir0909 commented on index.js:
```diff
@@ -61,7 +61,8 @@ function getUrlFromUsername( site, username ) {
         angellist: 'angel.co',
-        bitbucket: 'bitbucket.org'
+        bitbucket: 'bitbucket.org',
+        skype: 'none'
```
Instead of this, how about

```javascript
skype: 'skype'
```

And modify the switch statement to 
```javascript
case 'skype':
  return url_map[site] + ':' + username + '?call'
```

nihonjinrxs commented:
Works for me.  Will change and push the new commit.

nihonjinrxs commented:
Rebased to add merged commit, and fixed Skype icon fix per discussion.  Looks like there is still a merge conflict that needs to be resolved, but I'm not sure I can resolve it on my side.

nihonjinrxs commented:
Rebased to add merged pull request #37, and fixed Skype icon fix per discussion, but it still seems I've got a merge conflict.  Thoughts?

mudassir0909 commented:
Strange! I'm assuming you took a pull from `upstream/master` branch & did `git rebase upstream/master`

Can you try one more time ?

nihonjinrxs commented:
OK, I think we should be good now.  I had to merge my forked remote -- that was the problem.  Now things should work.
